substrate/frame/revive/src/lib.rs

```diff
@@ -1689,12 +1758,10 @@ impl<T: Config> Pallet<T> {
 		if tx.chain_id.is_none() {
 			tx.chain_id = Some(T::ChainId::get().into());
 		}
-		if tx.gas_price.is_none() {
-			tx.gas_price = Some(effective_gas_price);
-		}
-		if tx.max_priority_fee_per_gas.is_none() {
-			tx.max_priority_fee_per_gas = Some(effective_gas_price);
-		}
+
+		// tx.into_call expects tx.gas_price to be the effective gas price
+		tx.gas_price = Some(effective_gas_price);
+		tx.max_priority_fee_per_gas = Some(0.into());
```

Comment:
maybe we should add a comment here saying that we don't support priority_fee for now, since the tips needs to be calculated pre-dispatch, and we only know then the estimate not the real gas usage